apis/v1/gateway_types.go

```diff
@@ -606,6 +601,31 @@ type GatewayTLSConfig struct {
 	Options map[AnnotationKey]AnnotationValue `json:"options,omitempty"`
 }
 
+// GatewayTLSConfig specifies frontend tls configuration for gateway.
+type GatewayTLSConfig struct {
+	// defaultTLS specifies the default client certificate validation configuration
+	// for all Listeners handling HTTPS traffic, unless a per-port configuration
+	// is defined.
+	//
+	// support: Core
+	//
+	// +required
+	// <gateway:experimental>
+	DefaultTLS FrontendTLSValidation `json:"defaultTLS"`
```

Comment:
Want to avoid names that stutter (`tls.defaultTLS` -> `tls.default`)
```suggestion
	Default FrontendTLSValidation `json:"default"`
```

Reply:
Also recommend an intermediate struct here so we leave room for other forms of TLS config at this default level. So `tls.default.frontendValidation.mode` instead of `tls.default.mode`.

Reply:
done I change structs a bit PTAL